### PR TITLE
respect plp's 'showPWA' option in mobile integration launch

### PR DESCRIPTION
**Pull request by vadlis**

### CODE REVIEW

File: lib/net/integration_url_parameter.flow [hunk 1]
```diff
@@ -26,10 +28,24 @@ getDomOrUrlParameter(tag : string) -> string {
...
+getIntegrationLaunchOptions() -> Json {
+	parseJson(getDomOrUrlParameter("integrationOptions"));
+}
+
+canIntegrationShowPWA() {
+	if (getIsIntegrationLaunch()) {
+		options = getIntegrationLaunchOptions();
+		getJsonBoolField(options, "showPWA", false);
+	} else {
+		false;
```
Comment (vadlis) [inline lib/net/integration_url_parameter.flow:49]: Myabe it have to be 'true' when it's not an integration launch?

File: lib/net/integration_url_parameter.flow [hunk 2]
```diff
@@ -26,10 +28,24 @@ getDomOrUrlParameter(tag : string) -> string {
...
 
 getIntegrationLaunchId() -> int {
 	s2i(getLaunchIdString());
 }
+
+getIntegrationLaunchOptions() -> Json {
+	parseJson(getDomOrUrlParameter("integrationOptions"));
+}
+
+canIntegrationShowPWA() {
```
Comment (yegordovganich) [inline lib/net/integration_url_parameter.flow:44]:
```suggestion
checkIntegrationShowPwaParameter() -> bool {
```

File: lib/net/integration_url_parameter.flow [hunk 3]
```diff
@@ -8,6 +8,8 @@ export {
 	getIsIntegrationLaunch() -> bool;
 
 	getIntegrationLaunchId() -> int;	
+	getIntegrationLaunchOptions() -> Json;
+	canIntegrationShowPWA() -> bool;
```
Comment (yegordovganich) [inline lib/net/integration_url_parameter.flow:12]:
```suggestion
	checkIntegrationShowPwaParameter() -> bool;
```

File: lib/net/integration_url_parameter.flow (hunk 1 above)
Comment (yegordovganich) [reply lib/net/integration_url_parameter.flow:49]: !getIsIntegrationLaunch() || getJsonBoolField(getIntegrationLaunchOptions(), "showPWA", false)

**Comment by sfandersen** [comment]
Looks good.

**Pull request merged**